_glaredb/sql-commands/update.md

```diff
@@ -0,0 +1,57 @@
+---
+layout: default
+title: UPDATE
+parent: SQL commands
+---
+
+# UPDATE
+
+The SQL standard for `UPDATE` is large and powerful
+so that you can update rows from a table with precise
+control. GlareDB supports most of the [PostgreSQL `UPDATE`]
+syntax and will eventually have full support.
+
+## Syntax
+
+```sql
+UPDATE table_name
+    SET column = value_expr [, ...]
+    [ WHERE condition ]
+```
+
+| Field        | Description                                          |
+| ------------ | ---------------------------------------------------- |
+| `table_name` | Source table to update                               |
+| `column`     | Column name whose will be updated                    |
+| `value_expr` | An SQL expression that evaluates to the column value |
+| `condition`  | An SQL expression that evaluates to a boolean        |
+
+Do not include the table’s name in the specification of a target column.
+For example,
+
+```sql
+UPDATE table_name
+  SET table_name.col = 1
+```
+
+The above command is invalid.
```

Comment:
I think I'd prefer to remove this section in case someone copy-pastes it or doesn't read it carefully.

I think instead we can in the table do:

`table_name` | The fully-qualified table name to update (ex: `public.users`)
`column` | A column to update. Column is not qualified (ex: `name`).